bridge-adaptivity/module/views.py

```diff
@@ -52,15 +52,12 @@ def get_context_data(self, **kwargs):
     def get_content_courses():
         try:
             return get_available_courses()
-        except ObjectDoesNotExist as exc:
+        except HttpClientError as exc:
             log.error(
```

Comment:
Use exception type of the logging message and exc.message is not required to be added in that case.